client/src/pages/Profile.js

```diff
@@ -10,11 +10,10 @@ const ImageBlur = styled.div`
   background-image: url(${example});
   background-size: cover;
   background-repeat: no-repeat;
-  height: 40vh;
+  height: 40%;
   width: 100vw;
   filter: blur(4px);
-  top: 32px;
-  position: relative;
+  margin-top: 65px;
```

Comment:
do you need this because of the `header`? If so, you could have a flexbox taking care of the sizing and leave some `height` declarations out. Just a thought

Reply:
I will try, thanks 🐨